System Tests/Environment/CI.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+struct CI {
```

Comment:
I had anticipated this would be a text file in the *root* of the repo so that it can be consumed by the demo app and all test targets as opposed to a source file.  We can fix this later though if you prefer.

Reply:
I would structure this differently as well, but the only usable Bitrise extension (Swift Variable Injector) that allows us to inject custom configuration uses Swift files. You can find more details about the extension and its configuration here: https://medium.com/@lucianoalmeida1/continuous-integration-environment-variables-in-ios-projects-using-swift-f72e50176a91

Reply:
Aha, OK, I'll check those docs.  Thanks for sharing.